extensions/scarb-snforge-test-collector/src/compilation/test_collector/config.rs

```diff
@@ -55,19 +54,20 @@ impl From<TestExpectation> for ExpectedTestResult {
     }
 }
 
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
-pub enum RawForkConfig {
+#[derive(Debug, Clone, PartialEq, Serialize)]
+pub enum ForkConfigString {
```

Comment:
Why this name? I think `RawForkConfig` was okay for unparsed form.

Reply:
Actually you are right, idk why I changed it